js/sales.js

```diff
@@ -24,56 +24,119 @@
-            var newHourItem = document.createElement('li');
-            var hourSoldCookies = location.simulatedCookiesPerHour[i];
-            if(militaryHour < 12){
+        var hoursOpen = 14;
+        
+        var tableRow = document.createElement('tr');    
+        document.querySelector('thead').appendChild(tableRow);
+        var locationHeader = document.createElement('th');
+        locationHeader.textContent = 'Store Locations';
+        tableRow.appendChild(locationHeader);
+
+        for(var i = 0; i < hoursOpen; i++){
+            var newHourHead = document.createElement('th');
+            newHourHead.setAttribute('scope', 'col');
+            var militaryHour = openingHour + i;
+            if(militaryHour === 12){
+                newHourHead.textContent = '12:00pm';
+            } else if(militaryHour === 24 || militaryHour === 0){
+                newHourHead.textContent = '12:00am';
+            } else if(militaryHour < 12){
                 //1am-11am
-                newHourItem.textContent = militaryHour + 'am: ' + hourSoldCookies + " cookies";
-            } else if(militaryHour === 12){
-                newHourItem.textContent = '12pm: ' + hourSoldCookies + ' cookies';
-            } else if(militaryHour === 24){
-                newHourItem.textContent = '12am: ' + hourSoldCookies + ' cookies';
+                newHourHead.textContent = militaryHour + ':00am';
             } else {
                 //1pm-11pm
-                newHourItem.textContent = (militaryHour - standardTimeOffSet) + 'pm: ' + hourSoldCookies + ' cookies';
+                newHourHead.textContent = (militaryHour - standardTimeOffSet) + ':00pm';
+            }
+            tableRow.appendChild(newHourHead);
+        }
+        return tableRow;
+    }
+
+    CookieStore.prototype.render = function(){
+        var tableStoreRow = document.createElement('tr');
+        tableStoreRow.setAttribute('scope', 'row');
+        var tableStoreHead = document.createElement('th');
+        tableStoreHead.textContent = this.name;
+        document.querySelector('tbody').appendChild(tableStoreRow);
+        tableStoreRow.appendChild(tableStoreHead);
+
+        calculateSimulatedCookiesPerHour(this);
+        
+        for(var i = 0; i < this.simulatedCookiesPerHour.length; i++){
+            var tableStoreData = document.createElement('td');
+            tableStoreData.textContent = this.simulatedCookiesPerHour[i];
+            tableStoreRow.appendChild(tableStoreData);
+        }
+        return tableStoreRow;
+    }
+
+    function createTotalsFooterRow(){
+        var totalsRow = document.createElement('tr');
+        totalsRow.setAttribute('scope', 'row');
+        var totalsHead = document.createElement('th');
+        totalsHead.textContent = 'Totals';
+        document.querySelector('tfoot').appendChild(totalsRow);
+        totalsRow.appendChild(totalsHead);
+
+        for(var i = 0; i < cookieStoreLocations[0].simulatedCookiesPerHour.length; i++){
```

Comment:
This outer `for` loop is confusing - perhaps we should make a variable scoped to our `IIFE` called something like `openHours` that has a value of 14 or something like that. I looked at this for several minutes thinking it must be a bug, wondering why we were only referencing `cookieStoreLocations[0]`.